code/modules/mob/living/simple_animal/hostile/moghes.dm

```diff
@@ -0,0 +1,158 @@
+//animals from moghes
+/mob/living/simple_animal/hostile/biglizard
+	name = "plains tyrant"
+	desc = "The Skrazi, or 'plains tyrant', is an apex predator from the world of Moghes. Reaching up to fifteen feet in height, these beasts have earned a near-mythical reputation among the Unathi, with hunting a tyrant having been historically considered an elaborate form of suicide. Since the Contact War, these mighty predators are increasingly endangered due to loss of their habitats."
+	icon = 'icons/mob/biglizard.dmi'
+	icon_state = "biglizard"
+	icon_living = "biglizard"
+	icon_dead = "biglizard_dead"
+	speak_emote = list("hisses","roars")
+	emote_hear = list("hisses","grumbles","growls")
+	emote_see = list("hisses ferociously", "stomps")
+	emote_sounds = list('sound/effects/creatures/monstergrowl.ogg')
+	turns_per_move = 5
+	speak_chance = 5
+	meat_type = /obj/item/reagent_containers/food/snacks/xenomeat
```

Comment:
```
/obj/item/reagent_containers/food/snacks/xenomeat
desc = "A slab of green meat. Smells like acid."
reagents_to_add = list(/singleton/reagent/nutriment/protein = 6, /singleton/reagent/acid/polyacid = 6)
```
Is this intentional? I thought xenomeat is for like greimorians or xenomorphs or whatever
Does not sound like unathi could eat it if it has polyacid in it

Reply:
This is a holdover from the OG plains tyrant which I forgot to change, should be fixed to normal meat now.